docs/people-and-groups/managing.md

```diff
@@ -12,9 +12,13 @@ To start managing people, click on the **gear** icon > **Admin settings** > **Pe
 
 ## Creating an account
 
-To add a new person, click **Add person** in the upper right corner. You’ll be prompted to enter their name and email address.
+To add a new person, click **Invite someone** in the upper right corner. You’ll be prompted to enter their first and last names and their email address. 
 
-If you’ve already [configured Metabase to use email](../configuring-metabase/email.md), Metabase will send the new user an invite email. Otherwise, it’ll give you a temporary password that you’ll have to send to the person you’re inviting by hand.
+You can optionally add attributes to that user account, though you can add attributes to accounts at any time (as well as [via SSO](../people-and-groups/start.md#authentication)). Metabase uses attributes to create [data sandboxes](../permissions/data-sandboxes.md).
+
+Click **Create** to activate their account. **Once created, this account is considered active, even if the person you invited never signs in**. The account will remain active until you [deactivate the account](#deactivating-an-account). If you're on a paid Metabase plan, all active accounts will count toward your user account total (see [how billing works](a https://www.metabase.com/pricing/how-billing-works)).
```

Comment:
```suggestion
Click **Create** to activate an account. **An account becomes active once you click **Create**, even if the person never signs in**. The account remains active until you [deactivate the account](#deactivating-an-account). If you're on a paid Metabase plan, all active accounts will count toward your user account total (see [how billing works](a https://www.metabase.com/pricing/how-billing-works)).
```

Not sure if this makes a huge difference, but I was thinking about how one person can have multiple accounts (same email used for more than one instance). This section assumes (or implies?) that one person = one account.